evm_loader/program/src/entrypoint.rs

```diff
@@ -596,7 +596,68 @@ fn process_instruction<'a>(
             Ok(())
         },
 
+        EvmInstruction::ResizeStorageAccount => {
+            debug_print!("Execute ResizeStorageAccount");
+            let account_info = next_account_info(account_info_iter)?;
+            let code_account = next_account_info(account_info_iter)?;
+            let code_account_new = next_account_info(account_info_iter)?;
```

Comment:
It needs to check that the new code_account has a data size more than the previous.

Reply:
And it needs to check that code_account_new was created by the current operator.